app/views/admin/editorial_remarks/new.html.erb

```diff
@@ -1,21 +1,34 @@
-<% page_title "New Editorial Remark" %>
+<% content_for :page_title, "New editorial remark" %>
+<% content_for :title, "New editorial remark" %>
+<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @editorial_remark)) %>
 
-<div class="row">
-  <div class="col-md-8">
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
     <section class="document_page" id="<%= dom_id(@edition) %>">
-      <%= render partial: 'admin/editions/edition', locals: { edition: @edition } %>
+      <%= render 'admin/editions/edition', edition: @edition %>
     </section>
   </div>
 
-  <div class="col-md-4">
-    <span class="back">
-      back to <%= link_to 'document', admin_edition_path(@edition) %>
-    </span>
+  <div class="govuk-grid-column-one-third">
+    <p class="govuk-body">
+      back to <%= link_to 'document', admin_edition_path(@edition), class: "govuk-link" %>
+    </p>
 
     <%= form_for [:admin, @edition, @editorial_remark], url: admin_edition_editorial_remarks_path(@edition) do |feedback_form| %>
-      <%= feedback_form.errors %>
-      <%= feedback_form.text_area :body, rows: 20, label_text: "Remark" %>
-      <%= feedback_form.submit "Submit remark", class: "btn btn-default" %>
+      <%= render "govuk_publishing_components/components/textarea", {
+        label: {
+          text: "Remark",
+          bold: true,
+        },
+        name: "editorial_remark[body]",
+        id: "editorial_remark_body",
+        error_items: errors_for(@editorial_remark.errors, :body),
+        rows: 20,
+      } %>
+
+      <%= render "govuk_publishing_components/components/button", {
+        text: "Submit remark"
```

Comment:
This button should probably become 'Add internal note' – although you may choose to do that as part of the [Move Notes/History/Fact-checking tabs](https://trello.com/c/xiy8xs27/888-move-notes-history-fact-checking-tabs-to-the-design-system) card.